diff --git a/LAB02_proyectoEcomerce_ContexMapper.cml b/LAB02_proyectoEcomerce_ContexMapper.cml
index 8e0c6c2..65c4f6a 100644
--- a/LAB02_proyectoEcomerce_ContexMapper.cml
+++ b/LAB02_proyectoEcomerce_ContexMapper.cml
@@ -2,47 +2,35 @@ ContextMap TiendaEnLineaMapaContexto {
   type SYSTEM_LANDSCAPE
   state TO_BE
 
-  contains GestionPedidosContext
-
+  contains GestionProductosContext
 }
 
 Domain ComercioElectronico {
-  domainVisionStatement "Dominio principal que abarca todas las funcionalidades y procesos del sistema de comercio electrónico."
+  domainVisionStatement  "Visión general del dominio de comercio electrónico."
 
-Subdomain GestionPedidos {
+  Subdomain GestionProductos {
     type CORE_DOMAIN
-    domainVisionStatement "Subdominio responsable del proceso de compra en línea, incluyendo el carrito de compras, finalización de pedidos y seguimiento."
   }
-
 }
 
-BoundedContext GestionPedidosContext implements GestionPedidos {
+BoundedContext GestionProductosContext implements GestionProductos {
   type FEATURE
-  domainVisionStatement "Maneja todo el proceso de compra en línea, incluyendo el carrito de compras, finalización de pedidos y seguimiento."
+  domainVisionStatement "Gestiona el catálogo de productos, incluyendo la descripción, imágenes y manejo del inventario."
   
-  Aggregate CarritoCompras {
-    Entity ItemCarrito {
+  Aggregate CatalogoProductos {
+    Entity Producto {
       aggregateRoot
       String productoId
-      String nombreProducto
-      Double precioUnitario
-      int cantidad
+      String nombre
+      String descripcion
+      Double precio
+      int cantidadEnInventario
     }
-  }
-
-  Aggregate Pedidos {
-    Entity Pedido {
-      aggregateRoot
+    
+    Entity DescripcionProducto {
       String productoId
-      String clienteId
-      Date fechaPedido
-    }
-
-    Entity LineaPedido {
-      String productoId
-      int cantidad
-      Double precioUnitario
+      String descripcion
+      String imagen
     }
   }
-
 }
